pip_build.py

```diff
@@ -22,29 +22,28 @@
 import pathlib
 import re
 import shutil
+from pathlib import Path
 
 # Needed because importing torch after TF causes the runtime to crash
 import torch  # noqa: F401
 
 package = "keras"
 build_directory = "tmp_build_dir"
 dist_directory = "dist"
-to_copy = ["setup.py", "README.md"]
+to_copy = ["pyproject.toml", "README.md", "requirements-common.txt"]
```

Comment:
Why copy the requirements file? To note, actual public-facing requirements are listed in setup.py. The `requirements-common.txt` file cannot be used on its own (it's incomplete) and the other requirements files are dev-facing (deps needed to run CI, not needed to run the package)

Reply:
Ok, removed here as it doesn't seem to make any difference.